{"level":"error","message":"Unhandled rejection: TypeError: action.interactionobject.followUp is not a function\n    at Object.module.exports.ReplyToInteractionWithMessage (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:1114:34)\n    at Object.module.exports.RunAction (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:228:18)\n    at Object.module.exports.Event_Handle (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:143:24)\n    at Client.<anonymous> (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\bot.js:587:30)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","service":"user-service","timestamp":"2021-10-07 03:15:37 PM +1100"}
{"level":"error","message":"Unhandled rejection: TypeError: action.interactionobject.followUp is not a function\n    at Object.module.exports.ReplyToInteractionWithMessage (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:1114:34)\n    at Object.module.exports.RunAction (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:228:18)\n    at Object.module.exports.Event_Handle (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:143:24)\n    at Client.<anonymous> (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\bot.js:587:30)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","service":"user-service","timestamp":"2021-10-07 03:18:55 PM +1100"}
{"level":"error","message":"Unhandled rejection: TypeError: action.interactionobject.followUp is not a function\n    at Object.module.exports.ReplyToInteractionWithMessage (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:1114:34)\n    at Object.module.exports.RunAction (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:228:18)\n    at Object.module.exports.Event_Handle (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:143:24)\n    at Client.<anonymous> (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\bot.js:587:30)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","service":"user-service","timestamp":"2021-10-07 03:20:00 PM +1100"}
{"level":"error","message":"Unhandled rejection: TypeError: action.interactionobject.followUp is not a function\n    at Object.module.exports.ReplyToInteractionWithMessage (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:1114:34)\n    at Object.module.exports.RunAction (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:228:18)\n    at Object.module.exports.Event_Handle (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:143:24)\n    at Client.<anonymous> (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\bot.js:587:30)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","service":"user-service","timestamp":"2021-10-07 03:20:14 PM +1100"}
{"level":"error","message":"Unhandled rejection: TypeError: action.interactionobject.followUp is not a function\n    at Object.module.exports.ReplyToInteractionWithMessage (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:1114:34)\n    at Object.module.exports.RunAction (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:228:18)\n    at Object.module.exports.Event_Handle (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:143:24)\n    at Client.<anonymous> (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\bot.js:587:30)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","service":"user-service","timestamp":"2021-10-07 03:21:13 PM +1100"}
{"level":"error","message":"Unhandled rejection: TypeError: action.interactionobject.followUp is not a function\n    at Object.module.exports.ReplyToInteractionWithMessage (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:1114:34)\n    at Object.module.exports.RunAction (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:228:18)\n    at Object.module.exports.Event_Handle (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:143:24)\n    at Client.<anonymous> (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\bot.js:587:30)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","service":"user-service","timestamp":"2021-10-07 03:21:17 PM +1100"}
{"level":"error","message":"Unhandled rejection: TypeError: action.interactionobject.followUp is not a function\n    at Object.module.exports.ReplyToInteractionWithMessage (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:1114:34)\n    at Object.module.exports.RunAction (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:228:18)\n    at Object.module.exports.Event_Handle (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\Handlers\\Events.js:143:24)\n    at Client.<anonymous> (C:\\Users\\mnapo\\Documents\\Bot\\Gatekeeper\\BotFiles\\bot.js:587:30)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","service":"user-service","timestamp":"2021-10-07 03:21:50 PM +1100"}
